docs/release-notes/flink-1.10.md

```diff
@@ -84,6 +84,12 @@ customizable in the future. Users that experience issues related to scheduling
 can fallback to the legacy scheduler by setting `jobmanager.scheduler: legacy`
 in their `flink-conf.yaml`.
 
+#### Java 11 Support ([FLINK-10725](https://issues.apache.org/jira/browse/FLINK-10725))
```

Comment:
May also want to add that the existing Java 8 artifacts can also be used Java 11; i.e., users don't have to compile Flink themselves just to use it on java11.

Reply:
Question, does Java 11 support imply java 9 / 10 support?

Reply:
@sjwiesman I don't think so. I haven't tested it but we are building a multi release jar for Java 11:
https://github.com/apache/flink/blob/de7440e5b625ef609ce7ad6631abc507b8ca109d/flink-dist/pom.xml#L634

cc: @zentol